Gemfile.lock

```diff
@@ -57,15 +57,14 @@ GEM
       builder
       mime-types
       xml-simple
-    aws-sdk (2.0.29)
-      aws-sdk-resources (= 2.0.29)
-    aws-sdk-core (2.0.29)
+    aws-sdk (2.0.48)
```

Comment:
no, the Gemfile is setting the version rather loosely to `'~2'`, so I just ran `bundle update aws-sdk` for this patch update.

There was actually a 2.1 minor version released a couple weeks ago which fixes a bug that caused the milestone-parser tests to incorrectly make network connections to the EC2 instance metadata service (slow), so i'll test out updating to `2.1.2` instead of `2.0.48` in this PR